scripts/docker/ubuntu16.04-deb.Dockerfile

```diff
@@ -1,8 +0,0 @@
-FROM ubuntu:16.04
-
-ARG artifact_url=""
-ADD ${artifact_url} /tmp
-RUN apt-get update
-RUN apt-get install -y /tmp/*mongosh_*_amd64.deb
-RUN /usr/bin/mongosh --version
-ENTRYPOINT [ "mongosh" ]
```

Comment:
This file is unused, we dropped support for Ubuntu 16.04 when we decided to align platform support with the 5.0 server.